google/cloud/security/iam/explain/importer/importer.py

```diff
@@ -335,8 +338,13 @@ def _convert_group(self, forseti_group):
                                 type='group',
                                 member_name=forseti_group))
 
+    def _get_permissions_for_role(self, rolename):
+        """Returns permissions defined for that role name."""
```

Comment:
Hi Felix, can we start to document the args here?  Even better, would you mind to start to enable the pylint checks for this file by removing the above disables?  If not now, do you have any plan to take care of this?  Once again, in the future, this might become someone's else code.  :)